### ISSUE TITLE
PRF Extension support

### ISSUE THREAD

**Issue by fcorneli** [Yubico/java-webauthn-server#328]
I've had to add the PRF extension support kind of around the library.
See:
https://github.com/e-Contract/enterprise-jsf/blob/master/ejsf-taglib/src/main/java/be/e_contract/ejsf/component/webauthn/WebAuthnUtils.java

Would be nice if `java-webauthn-server` would support this PRF extension out of the box.

**Comment by emlun** [comment]
Yes, this is on the roadmap, but we have no concrete time plan for it yet.

**Comment by emlun** [comment]
Hi again! PRF extension support is now available in [pre-release `2.7.0-RC1`](https://github.com/Yubico/java-webauthn-server/releases/tag/2.7.0-RC1) (but please use [`2.7.0-RC2`](https://github.com/Yubico/java-webauthn-server/releases/tag/2.7.0-RC2) to get an additional bug fix).
